app/services/container_topology_service.rb

```diff
@@ -80,10 +80,14 @@ def entity_status(entity, kind)
     when 'VM', 'Host' then entity.power_state.capitalize
     when 'Node'
       condition = entity.container_conditions.first
-      if condition.name == 'Ready' && condition.status == 'True'
-        'Ready'
+      if condition.nil?
+        'Unknown'
       else
-        'NotReady'
+        if condition.name == 'Ready' && condition.status == 'True'
+          'Ready'
+        else
+          'NotReady'
+        end
```

Comment:
To be future-proof we could loop over all the conditions:

    ready_status = "Unknown"
    entity.container_conditions.each do |condition|
      if conditon.try(:name) == 'Ready' && condition.try(:status) == 'True'
        ready_status = 'Ready'
      else
        ready_status = 'NotReady'
      end
    end
    ready_status